q2_sample_classifier/tests/test_visualization.py

```diff
@@ -58,8 +58,7 @@ def setUp(self):
         self.md_vaw = md_vaw.get_column('Column')
         table_vaw = self.get_data_path('vaw.qza')
         self.table_vaw = qiime2.Artifact.load(table_vaw)
-        self.table_vaw, = feature_table.actions.filter_samples(
-            self.table_vaw, metadata=md_vaw)
+        self.table_vaw.save('/tmp/vaw')
```

Comment:
why are you saving this table now? This is not clear from the other changes in this commit.